webapp/package.json

```
@@ -23,32 +24,86 @@
     "@babel/preset-react": "7.0.0",
     "@babel/preset-typescript": "7.3.3",
     "@babel/runtime": "7.4.5",
+    "@types/enzyme": "^3.10.4",
```

Can we use exact versions by removing the `^` characters on these dependency versions, including existing dependencies?
```suggestion
    "@types/enzyme": "3.10.4",
```

---

Got it, will fix this

---

Do you mind applying this change to all dependencies in `package.json`, then run `npm install`?